requirements.txt

```diff
@@ -1,10 +1,13 @@
+cx_Oracle
 numpy>1.7,<1.12; python_version == '2.6'
 numpy>1.7,<1.12; python_version == '3.2'
 numpy>1.7,<1.12; python_version == '3.3'
 numpy>1.7,<1.15; python_version == '2.7'
 numpy>1.7; python_version >= '3.4'
+pandas
```

Comment:
@jsturdy should it be `pandas` or `python-pandas` here as in the `setup.cfg`?
same for `sqlalchemy`
Also, next line contains a typo: `steup_requires`->`setup_requires`